apps/judicial-system/web/src/components/Table/Table.tsx

```diff
@@ -159,7 +159,7 @@ const Table = <T extends object>(
                     'Header',
                   )} `}</Text>
                   <span>
-                    {props.sortableColumnIds?.includes(column.id) ? (
+                    {!column.disableSortBy ? (
```

Comment:
`sortableColumnIds` is used by `PassedCases` and this change overrides that use. It would be better to settle for one way to define sortable columns and either use `sortableColumnIds` or `disableSortBy`, not both.